RELEASE-NOTES.txt

```diff
@@ -5,6 +5,7 @@
 
 17.3
 -----
+* [**] Added transform block capability [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3321]
```

Comment:
I thought this was released in 17.2.

Reply:
.... Yes. I got confused because I'm currently working on releasing 17.3 and time got all wibbly wobbly.